<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QSplitter" name="splitter_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Request</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QComboBox" name="cbox_RequestTypes"/>
              </item>
              <item>
               <widget class="QLineEdit" name="le_URL"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_Send">
                <property name="text">
                 <string>Send</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSplitter" name="splitter_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <widget class="QTabWidget" name="tabs_RequestSettings">
               <property name="currentIndex">
                <number>-1</number>
               </property>
              </widget>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Response</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QSplitter" name="splitter_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <widget class="QListWidget" name="list_Responses"/>
                  <widget class="QTextEdit" name="te_ResponseJsons">
                   <property name="readOnly">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
